testnets/unificationtestnet/assetlist.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "unificationtestnet",
+  "assets": [
+    {
+      "description": "Staking and governance coin for the Unification Testnet",
+      "denom_units": [
+        {
+          "denom": "nund",
+          "exponent": 0
+        },
+        {
+          "denom": "FUND",
+          "exponent": 9
+        }
+      ],
+      "base": "nund",
+      "name": "Unification Network",
+      "display": "FUND",
+      "symbol": "FUND",
```

Comment:
I would recommend a `traces` to point back to mainnet's using type: `"test-mintage"`

Reply:
Thanks for the recommendations & guidance - I have made the requested changes.